sos/report/plugins/ovn_central.py

```diff
@@ -110,7 +110,7 @@ def setup(self):
             'ovn-sbctl get-connection',
         ]
 
-        schema_dir = '/usr/share/openvswitch'
+        schema_dir = '/usr/share/ovn'
```

Comment:
This has been set as `openvswitch` since the plugin was first introduced 4 years ago. There's been a decent amount of changes done to this plugin, especially around db collections, and this has never been brought up.

Is this a change for newer ovn deployments, or has this always been wrong? If newer deployments use `ovn` but `openvswitch` was (at some point) correct, then we'll want to support both for at least some amount of time.

Reply:
OVN has born as part of OVS so probably initially those references were required, but now ovn_central is exclusivelly a plugin for ovn deployments, and those deployments using openvswitch will not include this plugin enabled. In any case let me clean a little the plugin code to remove the code that is not neccesary with mind on OVN exclusivelly. thx

Reply:
I finally reconsider your comment after checking that some version (2.11) still has reference to openvswitch. It will not be the usual since as I indicated the plugin is for exclusive use in ovn environments, but let's keep backward compatibility.

I took the opportunity to correct the use of container name instead of container ID that was not getting the remote files in the calls to get_tables_from_schema since it requires a container name and not container ID.